Tests/XcodeGenKitTests/SchemeGeneratorTests.swift

```diff
@@ -336,6 +336,48 @@ class SchemeGeneratorTests: XCTestCase {
                 try expect(buildableReference?.blueprintName) == "ExternalTarget"
                 try expect(buildableReference?.referencedContainer) == "container:\(externalProject.string)"
             }
+
+            $0.it("generates scheme with remote runnable for watch") {
+                let xcscheme = try self.makeWatchScheme(appType: .watch2App, extensionType: .watch2Extension)
+                try expect(xcscheme.launchAction?.runnable).beOfType(XCScheme.RemoteRunnable.self)
```

Comment:
Can you add the opposite check elsewhere? To ensure we only make it a `RemoteRunnable` in this case, and a `BuildableProductRunnable` elsewhere?